README.md

```diff
@@ -1,33 +1,50 @@
-OpenCanary
-=================
+# OpenCanary
 Thinkst Applied Research
 
 ![opencanary logo](docs/logo.png)
 
-Overview
-----------
+## Overview
 
 In essence, OpenCanary creates a network honeypot allowing you to catch hackers before they fully compromise your systems. As a technical definition, OpenCanary is a daemon that runs several canary versions of services that alerts when a service is (ab)used.
 
-Features
-----------
+## Table of Contents
```

Comment:
This ordering looks odd to me. How about
- [Prerequisites](#prerequisites)

- [Features](#features)

- [Installation on Ubuntu](#installation-ubuntu)
- [Installation on OS X](#installation-os-x)
- [Installation using Git](#installation-git)
- [Running OpenCanary](#running-opencanary)
- [Samba setup for SMB service](#samba-setup-optional)

- [Docker Compose Usage](#docker-compose)
- [Docker Usage](#docker)

- [FAQ](#faq)
- [Contributing](#contributing)
- [Code of Conduct](#code-of-conduct)

And match this ordering in the main doc below.